includes/Article.php

```diff
@@ -509,7 +509,7 @@ public function view() {
 		while ( !$outputDone && ++$pass ) {
 			switch( $pass ) {
 				case 1:
-					wfRunHooks( 'ArticleViewHeader', array( &$this, &$outputDone, &$useParserCache ) );
+					wfRunHooks( 'ArticleViewHeader', array( $this, &$outputDone, &$useParserCache ) );
 					break;
 				case 2:
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this "switch/case" number of lines from 56 to at most 10, for example by extracting code into function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1151)